### ISSUE TITLE
can't set style to default

### ISSUE THREAD

**Issue by matthias-k** [lukelbd/proplot#234]
### Description

I would like to set the style back to matplotlib's defaults (i.e., no background color, no grid, etc.). According to the documentation (https://proplot.readthedocs.io/en/latest/configuration.html#proplot-settings), this should be possible with `plot.rc.update(style='default')`, but this command crashes.
### Steps to reproduce

```python
import proplot as plot
plot.rc.update(style='default')
```

**Expected behavior**: style set back to matplotlib default, i.e., no background color, no grid, etc

**Actual behavior**: the update command fails with
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-29-2290163ff701> in <module>
      1 import proplot as plot
      2 
----> 3 plot.rc.update(style='default')

/opt/conda/lib/python3.8/site-packages/proplot/config.py in update(self, *args, **kwargs)
    786         kw.update(kwargs)
    787         for key, value in kw.items():
--> 788             self.__setitem__(prefix + key, value)
    789 
    790     @docstring.add_snippets

/opt/conda/lib/python3.8/site-packages/proplot/config.py in __setitem__(self, key, value)
    330         a ProPlot :ref:`added setting <rc_proplot>`.
    331         """
--> 332         kw_proplot, kw_matplotlib = self._get_synced_params(key, value)
    333         rc_proplot.update(kw_proplot)
    334         rc_matplotlib.update(kw_matplotlib)

/opt/conda/lib/python3.8/site-packages/proplot/config.py in _get_synced_params(self, key, value)
    399         elif key == 'style':
    400             if value is not None:
--> 401                 kw_matplotlib, kw_proplot = _get_style_dicts(value, infer=True)
    402 
    403         # Cycler

ValueError: too many values to unpack (expected 2)
```

### Equivalent steps in matplotlib

doesn't apply

### Proplot version

0.6.4

**Comment by martijnende** [comment]
I have the same issue with `python==3.8`, `proplot==0.6.4` and `matplotlib==3.3.2`

**Comment by lukelbd** [comment]
This was fixed by #240
